hazelcast-client/src/test/java/com/hazelcast/client/ClientRegressionWithMockNetworkTest.java

```diff
@@ -593,17 +595,27 @@ public void stateChanged(LifecycleEvent event) {
     }
 
     @Test
-    public void testDeadlock_WhenDoingOperationFromLifecycleListener_and_NearCache() {
-        HazelcastInstance instance = hazelcastFactory.newHazelcastInstance();
-        final ClientConfig clientConfig = new ClientConfig();
-        final NearCacheConfig nearCacheConfig = new NearCacheConfig();
```

Comment:
I think this `NearCacheConfig` never did anything, since it was not configured with a name. So the client map never picked it up.